scripts/st2bootstrap-el8.template.sh

```diff
@@ -145,15 +153,21 @@ install_st2_dependencies() {
 }
 
 install_mongodb() {
-  # Add key and repo for the latest stable MongoDB (3.4)
-  sudo rpm --import https://www.mongodb.org/static/pgp/server-3.4.asc
-  sudo sh -c "cat <<EOT > /etc/yum.repos.d/mongodb-org-3.4.repo
-[mongodb-org-3.4]
+
+  # Need yum perl module enabled on RHEL 8
+  if [[ "$RHEL" == "1" ]]; then
```

Comment:
Just ran the install script and it's failing on this line with the following error:

```
Last metadata expiration check: 0:02:58 ago on Mon 10 Feb 2020 11:41:16 AM UTC.
Dependencies resolved.
===================================================================================================================================================================
 Package                                Architecture                          Version                                    Repository                           Size
===================================================================================================================================================================
Installing:
 crudini                                noarch                                0.9.3-1.el8                                epel                                 29 k

Transaction Summary
===================================================================================================================================================================
Install  1 Package

Total download size: 29 k
Installed size: 61 k
Downloading Packages:
crudini-0.9.3-1.el8.noarch.rpm                                                                                                      26 kB/s |  29 kB     00:01    
-------------------------------------------------------------------------------------------------------------------------------------------------------------------
Total                                                                                                                               11 kB/s |  29 kB     00:02     
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                                                                                                           1/1 
  Installing       : crudini-0.9.3-1.el8.noarch                                                                                                                1/1 
  Running scriptlet: crudini-0.9.3-1.el8.noarch                                                                                                                1/1 
  Verifying        : crudini-0.9.3-1.el8.noarch                                                                                                                1/1 

Installed:
  crudini-0.9.3-1.el8.noarch                                                                                                                                       

Complete!
main: line 520: RHEL: unbound variable
############### ERROR ###############
# Failed on Install st2 dependencies (MongoDB) #
#####################################
[vagrant@centos8 ~]$ 
```

Reply:
My above update will resolve this.

Reply:
I fixed this by adding `RHEL=0` if not RHEL.